requirements.txt

```diff
@@ -9,7 +9,7 @@ pytest~=7.1.2
 pytest-qt~=4.1.0
 
 # Development
-black~=22.3.0
+black~=23.1.0
```

Comment:
@ch-sa FYI black releases a new major version and that is used in the Github Action. Update black in requirements.txt to align with Github Action.

Reply:
https://github.com/psf/black/releases

Reply:
Good to know 👍